src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -138,6 +138,10 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 
   render() {
     const { order } = this.props
+    const isPickupAvailable = get(
+      this.props,
+      "order.lineItems.edges[0].node.artwork.pickup_available"
```

Comment:
One #minor note that I _think_ we try to avoid using `get` because we lose the ability to type-check.

Reply:
This is an edge-case example I think since its so many props deep :) 

I think for props that are like one or two or three dots in then it makes sense to check using like 

```js
const isPickupAvailable = order && order.lineItems && order.lineItems.edges[0]
```

but if we were to check an entire chain like this it would become less readable (i think).